cumulus_library/template_sql/templates.py

```diff
@@ -288,6 +289,34 @@ def get_is_table_not_empty_query(
         )
 
 
+def get_object_denormalize_query(
+    schema_name: str,
+    source_table: str,
+    source_id: str,
+    field: str,
+    field_config: dict,
+    target_table: str,
+):
+    """Generates a table by expanding a specified row element
```

Comment:
nit: "get object denormalize query" and "expanding a specified row element" is a lot of jargon, which I can see makes total sense for someone embedded here. But as a sometimes reviewer, it gave me pause.

The wording is fine, but could maybe like a tiny example in the docstring be appropriate? It seems to do a tiny flatten of a target field, yeah?

Reply:
take a look at this update and see what you think.

Reply:
Sure! :+1: